doc/tutorials/python-kdb.md

```diff
@@ -4,6 +4,22 @@
 
 When programming in Python it is possible to access the kdb database, changing values of existing keys, adding and deleting keys and a few other things.
 
+## Installation
+
+Either [build](https://www.libelektra.org/bindings/swig_python) the package or install from a repository.
+
+### Alpine Linux
+
+The [python bindings package](https://pkgs.alpinelinux.org/packages?name=py3-elektra&branch=edge&repo=testing) is only available in the testing repository (as of 2019-04-29).
+
+You'll have to use `alpine:edge`
+
+```sh
+echo "http://dl-cdn.alpinelinux.org/alpine/edge/testing" >> /etc/apk/repositories
```

Comment:
Please also add "docker run -it alpine:edge /bin/sh" before so that this works for more users.

Reply:
I added it.